extensions/machine-learning-services/src/modelManagement/deployedModelService.ts

```diff
@@ -92,16 +104,40 @@ export class RegisteredModelService {
 			}
 		}
 	}
+	private loadModelData(row: azdata.DbCellValue[]): RegisteredModel {
+		return {
+			id: +row[0].displayValue,
+			artifactName: row[1].displayValue,
+			title: row[2].displayValue,
+			description: row[3].displayValue,
+			version: row[4].displayValue,
+			created: row[5].displayValue
+		};
+	}
+
+	private async updateModel(model: RegisteredModel): Promise<RegisteredModel | undefined> {
+		let connection = await this.getCurrentConnection();
+		let updatedModel: RegisteredModel | undefined = undefined;
+		if (connection) {
+			const query = this.getUpdateModelQuery(connection.databaseName, model);
+			let result = await this._queryRunner.safeRunQuery(connection, query);
+			if (result && result.rows && result.rows.length > 0) {
```

Comment:
can the null-conditional operator be used here to simplify?

Reply:
you mean result?.rows?  I can replace that but still have to check for length

---
In reply to: [397443949](https://github.com/microsoft/azuredatastudio/pull/9609#discussion_r397443949) [](ancestors = 397443949)